test/http/http.ml

```diff
@@ -1,5 +1,7 @@
 open Stdune
 
+let server_auto_shutdown = 30.
+
```

Comment:
Isn't it better to have it in the `Server.t` type? It could be pass as a value to make with a fixed default value. WDYT?

Reply:
Indeed, it can still be modified later if we need to.